config/nucleo_bridge/controller_config.json

```diff
@@ -149,6 +149,6 @@
         "quadCPR": -9072.0,

```

Comment:
does the new wrist still have this as the CPR?

Reply:
forgot to change it, changing it to 1024 to reflect icd